lib/game_data.js

```diff
@@ -105,6 +105,14 @@ class GameData {
     */
     addAdditionalItemProperties(iteminfo) {
         if (!this.items_game || !this.items_game_cdn || !this.csgo_english) return;
+        
+        //get sticker codename/name
+        for(var i=0;i<iteminfo.stickers.length;i++){
+            if (iteminfo.stickers[i].sticker_id in this.items_game['items_game']['sticker_kits']) {
```

Comment:
Since we use `sticker_kits` so often, we might as well store it a reference to it around here so it'd become

```
// outside this loop, maybe line 109?
const sticker_kits = this.items_game.items_game.sticker_kits

// above code

if (sticker.sticker_id in sticker_kits)
```